src/server/routes/contract/extensions/erc721/read/signaturePrepare.ts

```diff
@@ -286,7 +285,7 @@ export async function erc721SignaturePrepare(fastify: FastifyInstance) {
           }
           parsedUid = uid;
         } else {
-          parsedUid = stringToHex(uid, { size: 32 });
+          parsedUid = `0x${createHash("sha256").update(uid).digest("hex")}`;
```

Comment:
is this always guaranteed to be 32 bytes?

Reply:
Yes sha256 will always generate 32 bytes (256 bit -> 32 bytes)

<img width="754" alt="image" src="https://github.com/user-attachments/assets/d38259fb-89b4-4eb1-b3c4-9d2b2505f3bc">